Kernel/Graphics/VMWare/Console.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Graphics/VMWare/Console.h>
+#include <Kernel/WorkQueue.h>
+
+namespace Kernel {
+
+constexpr static AK::Time refresh_interval = AK::Time::from_milliseconds(16);
+
+NonnullRefPtr<VMWareFramebufferConsole> VMWareFramebufferConsole::initialize(VMWareDisplayConnector& parent_display_connector)
+{
+    auto current_resolution = parent_display_connector.current_mode_setting();
+    return adopt_ref_if_nonnull(new (nothrow) VMWareFramebufferConsole(parent_display_connector, current_resolution)).release_nonnull();
```

Comment:
I think this can be simplified as

```suggestion
    return adopt_ref(*new (nothrow) VMWareFramebufferConsole(parent_display_connector, current_resolution));
```